StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -144,6 +153,96 @@ class PaymentSheetLinkAccount: PaymentSheetLinkAccountInfoProtocol {
         }
     }
 
+    func startVerification(completion: @escaping (Result<Bool, Error>) -> Void) {
+        guard case .requiresVerification = sessionState else {
+            DispatchQueue.main.async {
+                completion(.success(false))
+            }
+            return
+        }
+
+        guard let session = currentSession else {
+            assertionFailure()
```

Comment:
we should avoid assertionFailures and use stpAssert -- and potentially kick off an event (Can we verify it's caught by the `completion(.failure(...))` and an event is fired?)

(note: there is also `assertionFailure` calls in many of these functions)